tests/issues/issues.yaml

```diff
@@ -5658,3 +5658,148 @@ test_cases:
                 imm: 0
                 access: CS_AC_READ
           regs_read: [ ra ]
+
+  - input:
+      name: "mipsel64r6 lapc as alias of addiupc with details & real"
+      bytes: [ 0x19, 0x00, 0x40, 0xec ]
+      arch: "CS_ARCH_MIPS"
+      options: [ CS_MODE_MIPS64R6, CS_OPT_DETAIL, CS_OPT_DETAIL_REAL ]
+      address: 0x0
+    expected:
+      insns:
+      - asm_text: "lapc $v0, 0x64"
+        id: MIPS_INS_ADDIUPC
```

Comment:
Those ones can't be strings. They must be a numerical value. Because `cyaml` doesn't support mixed types fr fields.

Reply:
If you want to change this, and use `char *` here, don't forget to add the IDs to `test_mapping.h`.

Reply:
i'm fixing this.